php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -170,7 +170,7 @@ class NDB_BVL_Instrument extends NDB_Page
      */
     static function factory($instrument, $commentID, $page, $guarantee_exists = true)
     {
-        //include_once 'HTML/QuickForm.php';
+        include_once 'HTML/QuickForm.php';
```

Comment:
If the include statement is commented, there is an error even if HTML_QuickForm is installed